android/src/main/java/com/tailscale/ipn/App.kt

```diff
@@ -71,6 +72,7 @@ class App : UninitializedApp(), libtailscale.AppContext {
   val dns = DnsConfig()
   private lateinit var connectivityManager: ConnectivityManager
   private lateinit var app: libtailscale.Application
+  private lateinit var healthNotifier: HealthNotifier
```

Comment:
It doesn't look like this is referenced anywhere but if it *is*, optional >> lateinit...   Lots of NPE bugs in need of fixing due to when onCreate() runs...

Reply:
Done, good call